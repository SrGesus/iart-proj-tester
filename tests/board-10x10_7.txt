FD	FB	BD	LH	LV	VD	FC	VD	FE	VE
BB	LH	BB	VC	VC	BE	LV	BD	FE	LH
VE	FD	FB	FB	BD	BD	BC	BC	LH	VD
VD	BC	LV	LV	BC	FD	BD	VC	FB	FB
FC	FE	FE	VE	BD	VB	VB	BB	FD	LH
FB	BE	BD	BB	BB	FD	FE	LH	VD	BD
VD	BB	FD	FC	BD	BE	VC	BB	VD	LH
FD	VC	FD	BE	BE	FE	VD	BC	VE	LH
VD	FD	FE	BB	LH	FD	BD	FD	LV	FB
VB	LV	LH	VE	FD	FB	VB	VB	BD	FC
