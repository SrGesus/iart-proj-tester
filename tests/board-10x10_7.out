FB	FD	BB	LH	LH	VE	FD	VE	FD	VE
BD	LH	BE	VB	VE	BD	LH	BC	FE	LV
VD	FE	FC	FC	BD	BC	BB	BB	LH	VC
VB	BB	LH	LH	BE	FD	BE	VD	FE	FB
FC	FC	FD	VE	BD	VE	VD	BB	FE	LV
FD	BB	BB	BC	BE	FC	FB	LV	VB	BE
VB	BE	FC	FB	BD	BB	VC	BD	VC	LV
FC	VD	FE	BD	BE	FC	VB	BC	VE	LV
VB	FE	FD	BE	LV	FD	BE	FB	LV	FC
VD	LH	LH	VC	FC	FD	VC	VD	BC	FE
